dotCMS/src/main/java/com/dotmarketing/db/HibernateUtil.java

```diff
@@ -233,6 +234,56 @@ public static void delete(Object obj) throws DotHibernateException {
 		}
 	}
 
+	/**
+	 * Do a query, settings the parameters (if exists) and return a unique result
+	 * @param clazz
+	 * @param query
+	 * @param parameters
+	 * @param <T>
+	 * @return
+	 * @throws DotHibernateException
+	 * @throws HibernateException
+	 */
+	public static <T> T load(final Class<T> clazz, final String query,
+							 final Object ...parameters) throws DotHibernateException {
+
+		T result = null;
+		int index = 0;
+		Query hibernateQuery = null;
+
+		try {
+
+			hibernateQuery = getSession().createQuery(query).setCacheable(useCache);
+
+			for (Object parameter : parameters) {
+				hibernateQuery.setParameter(index++, parameter);
+			}
+
+			result         = (T)hibernateQuery.list().get(0);
+			hibernateQuery = null;

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "hibernateQuery". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)